system-parachains/asset-hubs/asset-hub-polkadot/src/xcm_config.rs

```diff
@@ -105,9 +105,8 @@ pub type LocationToAccountId = (
 	// Different global consensus parachain sovereign account.
 	// (Used for over-bridge transfers and reserve processing)
 	GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>,
```

Comment:
`ExternalConsensusLocationsConverterFor` below also covers this one, so let's remove it

```suggestion
```